azure_pipeline-global.yaml

```diff
@@ -20,16 +20,36 @@ trigger:
     - /environments/stg/stg.tfvars
     - /environments/test/test.tfvars
 
+resources:
+  repositories:
+    - repository: cnp-azuredevops-libraries
+      type: github
+      ref: refs/heads/DTSPO-1188/ado-templates
+      name: hmcts/cnp-azuredevops-libraries
+      endpoint: 'hmcts'
+
 variables:
   terraformVersion: 0.15.0
   agentPool: 'hmcts-cftptl-agent-pool'
   build: $(Build.BuildNumber)
   timeoutInMinutes: 300
-  isMaster: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
   action:
   product: 'cft-platform'
+  template: vars/input-variables.yaml@cnp-azuredevops-libraries
+  precheckServiceConnection: 'azurerm-sandbox'
+  precheckKeyvaultName: 'infra-vault-nonprod'
+  precheckKeyvaultSecret: 'azure-devops-token'
```

Comment:
none of these are re-used, so this just looks like noise, hardcode them in the job where they are required
```suggestion
```